### DISCUSSION TITLE
Siliconflow的API可以支持调用吗？

### DISCUSSION THREAD

**Discussion by RSK21X** [chengfengfengwang/lingo-link#15] [Q&A]
Siliconflow出了不少免费的开源模型，但是没办法配置成功。

https://siliconflow.cn/zh-cn/

感谢作者

**Comment by chengfengfengwang** [comment]
#6 completed

<img width="1383" alt="image" src="https://github.com/chengfengfengwang/lingo-link/assets/23279667/925657fd-75cf-4f7f-ad42-3607fb7babf2">

**Comment by molezz** [comment]
为什么edge稳定版里没这个apikey ， 更新到新版033 ………………
![image](https://github.com/chengfengfengwang/lingo-link/assets/695412/36223324-a18d-4436-ae07-c7027a0cfa08)

**Comment by chengfengfengwang** [comment]
> 为什么edge稳定版里没这个apikey ， 更新到新版033 ……………… ![image](https://private-user-images.githubusercontent.com/695412/343753681-36223324-a18d-4436-ae07-c7027a0cfa08.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0ODg4NTUsIm5iZiI6MTcxOTQ4ODU1NSwicGF0aCI6Ii82OTU0MTIvMzQzNzUzNjgxLTM2MjIzMzI0LWExOGQtNDQzNi1hZTA3LWM3MDI3YTBjZmEwOC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNjI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDYyN1QxMTQyMzVaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT02NjAxNmM5MzEzOTZhNGU4YjE0OWNhNWZmNzZlNTA1MDkxZTRjZTE3YmU0OTc3NGU2YjNjM2JlOTA0ZmI0YjdjJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.7bU9utvwJ28xhc9Cufr0V7j0q0uIo3G9PZLGsQnJfmQ)

模型输入框0.3.3没有加，0.3.4加上的，edge还在审核😅
不想等的话可以下载release

**Comment by molezz** [comment]
原来如此， edge就是慢， 我还想是不是我的版本特殊了， 再用了edge beta也是
耐心 等

**Comment by RSK21X** [comment]
触发图标提供了upload，但是我上传png或者svg等等好像都无法正常显示

**Comment by chengfengfengwang** [comment]
> 触发图标提供了upload，但是我上传png或者svg等等好像都无法正常显示

我刚刚发现需要登录才能上传成功。你是处于登录还是未登录上传的？

**Comment by chengfengfengwang** [comment]
> 触发图标提供了upload，但是我上传png或者svg等等好像都无法正常显示

已修复
